tests/omni_lock_rust/tests/misc.rs

```diff
@@ -669,6 +695,48 @@ pub fn sign_tx_by_input_group(
                         &identity,
                         None,
                     )
+                } else if config.id.flags == IDENTITY_FLAGS_EOS {
```

Comment:
Mabey it's the time to add a interface.

Reply:
Could you give an example how to add a interface?